ByBit.Net/Clients/BybitSocketClient.cs

```diff
@@ -68,6 +68,10 @@ public BybitSocketClient(BybitSocketClientOptions options) : base("Bybit", optio
             SpotStreamsV3 = AddApiClient(new BybitSocketClientSpotStreamsV3(log, options));
 
             CopyTrading = AddApiClient(new BybitSocketClientCopyTradingStreams(log, options));
+
+            DerivativesPublic = AddApiClient(new BybitSocketClientDerivativesPublicStreams(log, options));
```

Comment:
Thank you for the question. Yes, because they provided one API endpoint for public streams (both for Contracts and UMA), but different endpoints for private streams. As a result, I decided not to duplicate the code and use it simultaneously. Please respond if it's suitable for you?

Reply:
Alright